### PR TITLE
add RunAsUser/RunAsUserName test case

### PR DESCRIPTION
**Pull request by Helen-Xie**
part of #20 
please review it @Random-Liu @feiskyer @xlgao-zju 

Signed-off-by: Helen-Xie <yulinxie@zju.edu.cn>

### CODE REVIEW

File: pkg/validate/security_context.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+				RunAsUsername: userName,
+			},
+		},
+	}
+	return createContainer(rc, ic, containerConfig, podID, podConfig), expectedLogMessage
+}
+
+// verifyLogOutput verifies that the container output log is the same as the expected value.
+func verifyLogOutput(containerID string, expectedLogMessage []byte) {
+	cmd := exec.Command("sh", "-c", "docker logs "+containerID)
```
Comment (feiskyer) [inline pkg/validate/security_context.go]: Why `docker logs` here? We should get the logs via CRI here.

Comment (Helen-Xie) [inline pkg/validate/security_context.go]: So sorry. I will update it.

Comment (Helen-Xie) [inline pkg/validate/security_context.go]: update. I check the container output by execsync, ready for review. @feiskyer

**Comment by feiskyer** [comment]
LGTM

**Pull request merged**